desk/gembind.c

```diff
@@ -220,14 +115,13 @@ WORD appl_init(void)
     gb.gb_pintout = int_out;
     gb.gb_padrin = addr_in;
     gb.gb_padrout = addr_out;
-    return gem_if(APPL_INIT);
+    return gem_if(AES_CTRL_CODE(APPL_INIT, 0, 1, 0));
 }
 
 
 WORD appl_exit(void)
 {
-    gem_if(APPL_EXIT);
-    return TRUE;
+    return gem_if(AES_CTRL_CODE(APPL_EXIT, 0, 1, 0));
```

Comment:
> Functional change: previous implementation always returned TRUE.

Yes, but thats actually not quite right. Anyway, the return value from appl_exit() isn't used anywhere. Maybe it would be clearer to declare it as void, it's rather questioable anyway how to react in the caller should this function ever return FALSE. Reentering the main-loop does not seem to be an option.